enterprise/admin/views.py

```diff
@@ -506,7 +506,10 @@ def notify_enrolled_learners(cls, enterprise_customer, request, course_id, users
                 args=(course_id,),
             )
         course_name = course_details.get('title')
-        course_start = parse_lms_api_datetime(course_details.get('start'))
+        course_start = course_details.get('start')
+
+        if course_start:
```

Comment:
Would it be better to modify `parse_lms_api_datetime` such that it does not fail if it receives a None value.  Alternatively, a more "pythonic" way to handle this would be to use `try/except` -- the whole EAFP vs LBYL thing:

```
try:
    course_start = parse_lms_api_datetime(course_details.get('start'))
except <error type>:
    course_start = None
```